src/lib/routes/admin-api/user/pat.ts

```diff
@@ -61,8 +75,18 @@ export default class PatController extends Controller {
                 openApiService.validPath({
                     tags: ['Personal access tokens'],
                     operationId: 'createPat',
+                    summary: 'Create a new Personal Access Token.',
+                    description:
+                        'Creates a new [Personal Access Token](https://docs.getunleash.io/how-to/how-to-create-personal-access-tokens) for the current user.',
                     requestBody: createRequestSchema('patSchema'),
-                    responses: { 200: createResponseSchema('patSchema') },
+                    responses: {
+                        201: resourceCreatedResponseSchema('patSchema'),
+                        ...getStandardResponses(401, 403),
+                        404: {
+                            description:
+                                'Is returned when Personal Access Tokens are disabled.',
+                        },
```

Comment:
Oh, this is very interesting. Can you just use 404 for these for now? We have some changes coming to the standard responses with the new error type, so we probably want to find a way to handle this. Can you instead put this in the operation description?

Reply:
Yeah it's just a 404 for now. Should I just put the description in the general description of the method, and make the 404 a standardResponse?